### ISSUE TITLE
fail to parse yml file

### ISSUE THREAD

**Issue by ubinexy** [adeptex/whispers#47]
**Describe the bug**
We have a yaml file like this 

```
- name: set APP_NAME according to TALK_NAME
  set_fact:
    APP_NAME: 'ecs-"{{lookup('env', 'TALK_NAME')}}"'
```

**To Reproduce**
using whispers-2.1.0 to scan the yaml file will get following messages

```
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 4, column 5:
        APP_NAME: 'ecs-"{{lookup('env',  ... 
        ^
expected <block end>, but found '<scalar>'
```

**Comment by adeptex** [comment]
hey @ubinexy, thanks for reporting! I will take a look..

**Comment by adeptex** [comment]
hello again @ubinexy, i've checked it out.

it appears that the YML you submitted is in fact invalid. 

In `'ecs-"{{lookup('env', 'TALK_NAME')}}"'` you have:
1. string `'ecs-"{{lookup('`
2. value `env`
3. string `', '`
4. value `TALK_NAME`
5. string `')}}"'`

There is no parser error if you remove outside single quotes:
```yaml
- name: set APP_NAME according to TALK_NAME
  set_fact:
    APP_NAME: ecs-"{{lookup('env', 'TALK_NAME')}}"
```

What do you think?

**Comment by ubinexy** [comment]
Thanks for the advice. I will take a look

**Comment by ubinexy** [comment]
Hi @adeptex, I've mistaken the content of yaml file

The original content in the file is like this:

```
- name: set APP_NAME according to TALK_NAME
  set_fact:
    APP_NAME: "ecs-{{lookup('env', 'TALK_NAME')}}"
```
but using whispers-2.1.0 to scan the yaml file will get following messages

```
  File ".../venv/lib/python3.8/site-packages/yaml/parser.py", line 438, in parse_block_mapping_key
    raise ParserError("while parsing a block mapping", self.marks[-1],
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 4, column 5:
        APP_NAME: 'ecs-"{{lookup('env',  ... 
        ^
expected <block end>, but found '<scalar>'
  in "<unicode string>", line 4, column 31:
        APP_NAME: 'ecs-"{{lookup('env', 'TALK_NAME')}}"'
                                  ^
[]

```

**Comment by adeptex** [comment]
hey @ubinexy, i confirm the bug. interesting case.. will have a look at patching it :+1:

**Comment by adeptex** [comment]
Bug fixed in version 2.1.1 - https://pypi.org/project/whispers/2.1.1/

@ubinexy thanks again for reporting! Please feel free to open another issue if the bug is not completely resolved.

cheers!
